src/main/scala/com/raphtory/core/components/graphbuilder/GraphBuilder.scala

```diff
@@ -9,6 +9,69 @@ import com.typesafe.config.Config
 import scala.collection.mutable
 import scala.collection.mutable.ArrayBuffer
 
+/**
+  * {s}`GraphBuilder`
+  *
+  * : `GraphBuilder` trait for creating a Graph by adding and deleting vertices and edges.
+  *    While using `GraphBuilder`, override {s}`parseTuple(tuple: T)` to define parsing of rows and use methods {s}`addVertex`, {s}`addEdge`
```

Comment:
this also creates a bunch of updates, we should also explain these. what does a  new update represent, where is it stored, when are they used, who retrieves and processes them? 

missing the docs for the getUpdates function, what does that do ?